tldw_Server_API/Config_Files/.env.example

```
@@ -1,222 +1,191 @@
+# OPTIONAL — MCP Security
 # ================================================
-# For remote vector DB services
-VECTOR_DB_API_KEY=
+# Auto-generated by Docker entrypoint. Set explicitly for shared/production deployments.
+# Min 32 chars each.
 
-# In production environments, set:
-# tldw_production=true
-# SHOW_API_KEY_ON_STARTUP=false
+MCP_JWT_SECRET=CHANGE_ME_TO_SECURE_MCP_JWT_SECRET
+MCP_API_KEY_SALT=CHANGE_ME_TO_SECURE_MCP_API_KEY_SALT
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MCP_API_KEY_SALT key should go before the MCP_JWT_SECRET key

(UnorderedKey)